### PR TITLE
Ingest binary and test-runner package from common

### PR DESCRIPTION
**Pull request by dmitrii-ubskii**
## Usage and product changes

We bring in the binary package and test-runner from typedb-common. `typedb-runner` is deployed to maven such that we can safely depend on it from other repos without creating Bazel dependency cycles.

### CODE REVIEW

File: dependencies/vaticle/repositories.bzl [hunk 1]
```diff
@@ -27,8 +27,8 @@ def vaticle_bazel_distribution():
 def vaticle_dependencies():
     git_repository(
         name = "vaticle_dependencies",
-        remote = "https://github.com/vaticle/dependencies",
-        commit = "371a67999d92b4d0833cd5e016ec01c6278e72c4", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
+        remote = "https://github.com/dmitrii-ubskii/dependencies",
+        commit = "f7f66a4353bd7727f49b513715b05e221c9e6ed6", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
```
Comment (dmitrii-ubskii) [inline dependencies/vaticle/repositories.bzl:31]: Note

File: dependencies/vaticle/repositories.bzl [hunk 2]
```diff
@@ -41,8 +41,8 @@ def vaticle_typeql():
 def vaticle_typedb_common():
     git_repository(
         name = "vaticle_typedb_common",
-        remote = "https://github.com/vaticle/typedb-common",
-        tag = "2.26.0",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_common
+        remote = "https://github.com/dmitrii-ubskii/typedb-common",
+        commit = "fffbf734614068078655248032f27270aeca26d1",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_common
```
Comment (dmitrii-ubskii) [inline dependencies/vaticle/repositories.bzl:45]: Note

File: WORKSPACE [hunk 3]
```diff
@@ -155,7 +155,7 @@ vaticle_typedb_console_artifact()
 load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_artifacts = "artifacts")
 load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_artifacts = "artifacts")
 load("@vaticle_typedb_protocol//dependencies/maven:artifacts.bzl", vaticle_typedb_protocol_artifacts = "artifacts")
-load("//dependencies/maven:artifacts.bzl", vaticle_typedb_artifacts = "artifacts")
+load("//dependencies/maven:artifacts.bzl", vaticle_typedb_artifacts = "artifacts", vaticle_typedb_internal_artifacts = "internal_artifacts")
```
Comment (flyingsilverfin) [inline WORKSPACE:158]: These should be moved to 'vaticle/artifacts.bzl' again and also call it `vaticle_typedb_vaticle_maven_artifacts`

File: dependencies/maven/artifacts.snapshot [hunk 4]
```diff
@@ -32,6 +32,8 @@
 @maven//:com_googlecode_java_diff_utils_diffutils_1_3_0
 @maven//:com_squareup_okhttp_okhttp_2_7_5
 @maven//:com_squareup_okio_okio_1_17_5
+@maven//:com_vaticle_typedb_typedb_common_7fd4316e7fd4c3ba9ce94f2ea73f12c82057930b
```
Comment (flyingsilverfin) [inline dependencies/maven/artifacts.snapshot:35]: depending on Common via maven artifact should not be happening anymore

File: server/parameters/util/OptionParser.java [hunk 5]
```diff
@@ -131,7 +131,7 @@ public Optional<Integer> parse(Set<Option> options) {
             Optional<Option> option = matchingOptions(options).first();
             if (option.isEmpty()) return Optional.empty();
             else if (!option.get().hasValue())
-                throw TypeDBException.of(CLI_OPTION_REQUIRES_TYPED_VALUE, option.get());
+                throw TypeDBException.of(CLI_OPTION_REQUIRES_TYPED_VALUE, option.get(), valueDescription);
```
Comment (flyingsilverfin) [inline server/parameters/util/OptionParser.java:134]: nice catch

File: dependencies/vaticle/repositories.bzl [hunk 6]
```diff
@@ -27,11 +27,10 @@ def vaticle_bazel_distribution():
 def vaticle_dependencies():
     git_repository(
         name = "vaticle_dependencies",
-        remote = "https://github.com/vaticle/dependencies",
-        commit = "d5cef8548663902c033edc5d706dbe9405e3ad7d",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
+        remote = "https://github.com/dmitrii-ubskii/dependencies",
```
Comment (flyingsilverfin) [inline dependencies/vaticle/repositories.bzl:30]: Don't forget to bump all these

**Review by flyingsilverfin — COMMENTED** [review]
See comment - bump deps and update console to not depend on 'common'

**Review by flyingsilverfin — APPROVED** [review]

**Pull request merged**